chia/wallet/trading/trade_store.py

```diff
@@ -246,6 +247,30 @@ async def get_all_trades(self) -> List[TradeRecord]:
 
         return records
 
+    async def get_trades_between(
+        self, start: int, end: int, sort_key: Optional[str] = None, reverse: bool = False
```

Comment:
`reverse` isn't being used

Reply:
Hmm yes I suppose it would be wise to add some tests :)